_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -763,372 +763,2410 @@ Terraform::
   link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library[How to Use the Gruntwork
   Infrastructure as Code Library].
 
+Keybase (optional)::
+  As part of this guide, you will create IAM users, including, optionally, credentials for those IAM users. If you
+  choose to create credentials, those credentials will be encrypted with a PGP key. You could provide the PGP keys
+  manually, but a more manageable option may be to have your team members to sign up for https://keybase.io[Keybase],
+  create PGP keys for themselves, and then you can provide their Keybase usernames, and the PGP keys will be retrieved
+  automatically.
+
 [[gruntwork_solution]]
 === The Gruntwork solution
 Gruntwork offers battle-tested infrastructure as code modules to help you create production grade infrastructure in a
 fraction of the time it would take to develop from scratch. Each of the code modules are "compliance-ready"; they are
 mostly unopinionated by default, but they can be configured for compliance with the right settings.
 
-To further simplify and expedite compliance, the Gruntwork Compliance modules are "wrappers" around the core,
-unopinionated modules in the Infrastructure as Code Library. The wrappers call the core modules with configuration values
-that are compliant with the AWS Foundations Benchmark. You can use the wrapper modules by creating a module of your own
-(this can be considered a second wrapper) and using the compliance module as the `source`. Optionally, you can also use
-`terragrunt` to call your module, thus creating a chain of IaC modules.
- +
- +
+To further simplify and expedite compliance, the Gruntwork CIS Service Catalog uses both standalone compliance
+modules as well as "wrappers" around the core, unopinionated modules in the Infrastructure as Code Library. The
+standalone compliance modules are designed with the compliance requirements built-in, whereas the wrappers
+call the core modules with configuration values that are compliant with the AWS Foundations Benchmark. You can use both
+types of modules by creating a module of your own (in the case of the wrapper modules, this can be considered a second
+wrapper) and using the compliance module as the `source`. Optionally, you can also use `terragrunt` to call your module,
+thus creating a chain of IaC modules. For more information on creating your own service catalog of modules please refer
+to the link:https://github.com/gruntwork-io/terraform-aws-service-catalog/blob/master/core-concepts.md#create-your-own-service-catalog[Create your own service catalog section]
+in the Gruntwork Service Catalog documentation.
+ +
+ +
+
+.Types of CIS module relationships to avoid repetitive code and minimize the amount of extra work needed to achieve compliance.
+image::/assets/img/guides/cis-compliance/cis-module-relationships.png[]
```

Comment:
nice diagram @robmorgan!

Reply:
I'm not sure if users will understand that the `infrastructure-live` repo is optional because this walkthrough is written entirely using that repo. Hmm..

Reply:
I suppose the section below, which talks about how to Create your own service catalog, is sufficient to clarify this point.

Reply:
@rhoboat ok!